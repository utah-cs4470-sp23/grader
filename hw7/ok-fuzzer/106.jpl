assert (((sum[a : (array[a : argnum, b : argnum, c : 1] argnum)[(sum[a : 441, b : 835, c : argnum] c), (- argnum), args[862]], b : (sum[a : argnum, b : argnum, c : (argnum + 231)] a), c : (- (- argnum))] (- c)) - (if (argnum <= 504) then {argnum, 15.87} else {256, 41.59}){0}) != (sum[a : (((402 - argnum) / (949 % 127)) + (- 16)), b : 975, c : (sum[a : (- (741 - argnum))] (sum[b : (argnum / argnum), c : (817 - 867), d : argnum] c))] {[c, 751, b], (! false), (sum[d : a, e : argnum, f : a] f)}{2})), "a"
let {} = (if (((24.85 + 43.2) == 96.69) || ((- 11.66) > 43.78)) then (array[a : (if true then 302 else 78), b : argnum, c : args[argnum]] (array[d : b, e : argnum] {}))[(sum[a : argnum] a), argnum, (sum[a : argnum] a)] else (array[a : 250, b : (if true then 285 else argnum)] {{}, b, args}{0}))[(array[a : (sum[a : 785] argnum)] (array[b : 504, c : a, d : 50] argnum))[argnum][argnum, (662 % [430, argnum][399]), (array[a : (sum[a : 188, b : 7] argnum), b : (if false then 677 else argnum), c : argnum] b)[(- 137), (sum[a : (133 / argnum), b : 262, c : 609] 850), 754]], (sum[a : {(- argnum)}{0}] (sum[b : ((sum[b : 223, c : 729, d : a] argnum) / (- argnum)), c : argnum, d : {a, 556}{0}] [319][(- argnum)]))]
time let {a[b, c, d]} = (if (! (if true then (80.98 != 76.92) else false)) then (if (44 > (if false then argnum else 360)) then {(array[a : 172, b : 887, c : argnum] argnum)} else {(array[a : argnum, b : 793, c : 698] c)}) else {(array[a : (sum[a : argnum, b : 885] b), b : (if false then argnum else argnum), c : argnum] (sum[d : 73] argnum))})

read image "e.png" to e

print "f"

read image "f.png" to f

let {g[h, i]} = (if (if (true && (array[g : b, h : b, i : d] false)[(sum[g : c, h : c] g), (- argnum), b]) then (array[g : (sum[g : 80, h : d] h), h : c] (false || false))[d, argnum] else (((46.43 < 15.49) || (! false)) && (array[g : 394, h : 541] true)[(- argnum), a[609, d, b]])) then (if ((false && true) || ((! false) && (true || false))) then {(array[g : 241, h : c, i : 487] f)[c, (- 211), (- 996)]} else {{e}}{0}) else {(if (18.04 <= (- 68.5)) then (array[g : b] e)[(155 - argnum)] else f)})
let j[k, l, m] = (array[j : b, k : (array[j : h] (if false then a else a))[h][argnum, {(- i), 954}{0}, d], l : b] true)
show 14

let n = m
let {o[p, q], r[s, t]} = (if {[args], (array[o : h] false)}{1}[c] then (if (m != 293) then (array[o : d, p : (- i), q : b] {e, e})[(c + c), (l % argnum), k] else {f, f}) else {(if (argnum != (k + l)) then f else e), f})
time let {u} = {((- (- 68.64)) != (array[u : h, v : b] 1.87)[m, n])}

