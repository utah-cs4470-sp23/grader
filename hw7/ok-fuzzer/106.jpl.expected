(TimeCmd (ReadCmd "a.png" (VarArgument a)))
(LetCmd
 (ArgLValue (ArrayArgument b c d e))
 (ArrayLoopExpr
  (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
  b
  (SumLoopExpr
   (IntType)
   b
   (UnopExpr (IntType) - (VarExpr (IntType) argnum))
   (SumLoopExpr
    (IntType)
    c
    (IntExpr (IntType) 950)
    d
    (ArrayIndexExpr
     (IntType)
     (ArrayLoopExpr
      (ArrayType (IntType) 3)
      c
      (VarExpr (IntType) argnum)
      d
      (VarExpr (IntType) argnum)
      e
      (IntExpr (IntType) 341)
      (VarExpr (IntType) d))
     (IntExpr (IntType) 207)
     (IntExpr (IntType) 672)
     (BinopExpr (IntType) (IntExpr (IntType) 293) * (VarExpr (IntType) b)))
    e
    (ArrayIndexExpr
     (IntType)
     (VarExpr (ArrayType (IntType) 1) args)
     (VarExpr (IntType) argnum))
    (BinopExpr (IntType) (VarExpr (IntType) b) * (VarExpr (IntType) argnum))))
  c
  (IntExpr (IntType) 871)
  d
  (IntExpr (IntType) 71)
  (ArrayIndexExpr
   (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
   (IfExpr
    (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
    (TupleIndexExpr
     (BoolType)
     (TupleLiteralExpr
      (TupleType
       (BoolType)
       (BoolType)
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2))
      (TrueExpr (BoolType))
      (FalseExpr (BoolType))
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       a))
     0)
    (VarExpr
     (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
     a)
    (VarExpr
     (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
     a))
   (VarExpr (IntType) c)
   (ArrayIndexExpr
    (IntType)
    (ArrayLoopExpr
     (ArrayType (IntType) 3)
     e
     (VarExpr (IntType) c)
     f
     (VarExpr (IntType) d)
     g
     (IntExpr (IntType) 792)
     (VarExpr (IntType) b))
    (SumLoopExpr
     (IntType)
     e
     (UnopExpr (IntType) - (IntExpr (IntType) 671))
     (IntExpr (IntType) 337))
    (IntExpr (IntType) 826)
    (TupleIndexExpr
     (IntType)
     (TupleLiteralExpr
      (TupleType
       (IntType)
       (FloatType)
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2))
      (VarExpr (IntType) argnum)
      (FloatExpr (FloatType) 23)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       a))
     0)))))
(LetCmd
 (TupleLValue
  (TupleLValue
   (TupleLValue)
   (TupleLValue
    (ArgLValue (VarArgument f))
    (ArgLValue (VarArgument g))
    (ArgLValue (VarArgument h))))
  (ArgLValue (VarArgument i))
  (ArgLValue (VarArgument j)))
 (ArrayIndexExpr
  (TupleType
   (TupleType
    (TupleType)
    (TupleType (IntType) (ArrayType (IntType) 1) (BoolType)))
   (FloatType)
   (FloatType))
  (ArrayLoopExpr
   (ArrayType
    (TupleType
     (TupleType
      (TupleType)
      (TupleType (IntType) (ArrayType (IntType) 1) (BoolType)))
     (FloatType)
     (FloatType))
    1)
   f
   (SumLoopExpr
    (IntType)
    f
    (IfExpr
     (IntType)
     (TrueExpr (BoolType))
     (UnopExpr (IntType) - (VarExpr (IntType) c))
     (VarExpr (IntType) c))
    g
    (VarExpr (IntType) e)
    (VarExpr (IntType) argnum))
   (TupleLiteralExpr
    (TupleType
     (TupleType
      (TupleType)
      (TupleType (IntType) (ArrayType (IntType) 1) (BoolType)))
     (FloatType)
     (FloatType))
    (TupleLiteralExpr
     (TupleType
      (TupleType)
      (TupleType (IntType) (ArrayType (IntType) 1) (BoolType)))
     (TupleLiteralExpr (TupleType))
     (TupleLiteralExpr
      (TupleType (IntType) (ArrayType (IntType) 1) (BoolType))
      (IntExpr (IntType) 731)
      (VarExpr (ArrayType (IntType) 1) args)
      (FalseExpr (BoolType))))
    (CallExpr (FloatType) sin (FloatExpr (FloatType) 54))
    (CallExpr
     (FloatType)
     acos
     (CallExpr (FloatType) acos (FloatExpr (FloatType) 81)))))
  (BinopExpr (IntType) (IntExpr (IntType) 482) / (VarExpr (IntType) d))))
(LetCmd
 (ArgLValue (ArrayArgument k l m n))
 (VarExpr
  (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
  b))
(LetCmd
 (TupleLValue
  (TupleLValue
   (TupleLValue
    (ArgLValue (ArrayArgument o p q r))
    (ArgLValue (VarArgument s)))
   (ArgLValue (VarArgument t)))
  (ArgLValue (ArrayArgument u v)))
 (IfExpr
  (TupleType
   (TupleType
    (TupleType
     (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
     (BoolType))
    (IntType))
   (ArrayType (IntType) 1))
  (UnopExpr
   (BoolType)
   !
   (BinopExpr
    (BoolType)
    (ArrayIndexExpr
     (BoolType)
     (ArrayLiteralExpr (ArrayType (BoolType) 1) (VarExpr (BoolType) h))
     (VarExpr (IntType) l))
    ||
    (BinopExpr (BoolType) (VarExpr (IntType) l) != (VarExpr (IntType) argnum))))
  (TupleLiteralExpr
   (TupleType
    (TupleType
     (TupleType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
      (BoolType))
     (IntType))
    (ArrayType (IntType) 1))
   (TupleLiteralExpr
    (TupleType
     (TupleType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
      (BoolType))
     (IntType))
    (TupleLiteralExpr
     (TupleType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
      (BoolType))
     (VarExpr
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
      b)
     (VarExpr (BoolType) h))
    (VarExpr (IntType) n))
   (IfExpr
    (ArrayType (IntType) 1)
    (UnopExpr
     (BoolType)
     !
     (BinopExpr (BoolType) (VarExpr (IntType) m) != (IntExpr (IntType) 647)))
    (VarExpr (ArrayType (IntType) 1) args)
    (VarExpr (ArrayType (IntType) 1) g)))
  (TupleLiteralExpr
   (TupleType
    (TupleType
     (TupleType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
      (BoolType))
     (IntType))
    (ArrayType (IntType) 1))
   (IfExpr
    (TupleType
     (TupleType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
      (BoolType))
     (IntType))
    (IfExpr
     (BoolType)
     (BinopExpr (BoolType) (VarExpr (IntType) d) != (VarExpr (IntType) e))
     (BinopExpr (BoolType) (VarExpr (IntType) l) >= (VarExpr (IntType) l))
     (TrueExpr (BoolType)))
    (TupleLiteralExpr
     (TupleType
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (BoolType))
      (IntType))
     (TupleLiteralExpr
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (BoolType))
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       k)
      (TrueExpr (BoolType)))
     (VarExpr (IntType) e))
    (TupleLiteralExpr
     (TupleType
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (BoolType))
      (IntType))
     (TupleLiteralExpr
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (BoolType))
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       b)
      (TrueExpr (BoolType)))
     (VarExpr (IntType) m)))
   (TupleIndexExpr
    (ArrayType (IntType) 1)
    (IfExpr
     (TupleType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
      (ArrayType (IntType) 1)
      (IntType))
     (BinopExpr (BoolType) (VarExpr (BoolType) h) || (VarExpr (BoolType) h))
     (TupleLiteralExpr
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (ArrayType (IntType) 1)
       (IntType))
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       b)
      (VarExpr (ArrayType (IntType) 1) args)
      (VarExpr (IntType) argnum))
     (TupleLiteralExpr
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (ArrayType (IntType) 1)
       (IntType))
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       b)
      (VarExpr (ArrayType (IntType) 1) g)
      (IntExpr (IntType) 292)))
    1))))
(LetCmd
 (ArgLValue (ArrayArgument w x y))
 (TupleIndexExpr
  (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)
  (TupleIndexExpr
   (TupleType
    (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
    (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2))
   (IfExpr
    (TupleType
     (TupleType
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
      (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 2)))
    (TrueExpr (BoolType))
    (TupleLiteralExpr
     (TupleType
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)))
     (TupleLiteralExpr
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2))
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       o)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       a)))
    (TupleLiteralExpr
     (TupleType
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)))
     (TupleLiteralExpr
      (TupleType
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2))
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        3)
       k)
      (VarExpr
       (ArrayType
        (TupleType (FloatType) (FloatType) (FloatType) (FloatType))
        2)
       a))))
   0)
  1))
(AssertCmd (VarExpr (BoolType) s) "z")
(LetCmd (ArgLValue (VarArgument z)) (VarExpr (ArrayType (IntType) 1) u))
(LetCmd
 (ArgLValue (VarArgument A))
 (VarExpr
  (ArrayType (TupleType (FloatType) (FloatType) (FloatType) (FloatType)) 3)
  o))
(ReadCmd "B.png" (VarArgument B))
(LetCmd
 (TupleLValue (ArgLValue (VarArgument C)))
 (ArrayIndexExpr
  (TupleType (BoolType))
  (ArrayIndexExpr
   (ArrayType (TupleType (BoolType)) 2)
   (ArrayLoopExpr
    (ArrayType (ArrayType (TupleType (BoolType)) 2) 1)
    C
    (ArrayIndexExpr
     (IntType)
     (ArrayLoopExpr
      (ArrayType (IntType) 2)
      C
      (VarExpr (IntType) r)
      D
      (VarExpr (IntType) f)
      (VarExpr (IntType) r))
     (VarExpr (IntType) r)
     (VarExpr (IntType) l))
    (ArrayLoopExpr
     (ArrayType (TupleType (BoolType)) 2)
     D
     (VarExpr (IntType) x)
     E
     (VarExpr (IntType) l)
     (TupleLiteralExpr (TupleType (BoolType)) (VarExpr (BoolType) h))))
   (UnopExpr (IntType) - (UnopExpr (IntType) - (VarExpr (IntType) y))))
  (VarExpr (IntType) argnum)
  (IfExpr
   (IntType)
   (TrueExpr (BoolType))
   (VarExpr (IntType) y)
   (VarExpr (IntType) p))))
(AssertCmd (TrueExpr (BoolType)) "D")
